pkg/scheduler/plugins/coscheduling/core/core.go

```diff
@@ -65,6 +65,7 @@ type Manager interface {
 	ActivateSiblings(*corev1.Pod, *framework.CycleState)
 	AllowGangGroup(*corev1.Pod, framework.Handle, string)
 	Unreserve(context.Context, *framework.CycleState, *corev1.Pod, string, framework.Handle, string)
+	GetGangByGangId(gangId string) *Gang
```

Comment:
rename to `GetGang(namespace, name string) *Gang`

Reply:
ok